### ISSUE TITLE
Mov doenst allow strings

### ISSUE THREAD

**Issue by TheProxyRE** [x64dbg/x64dbg#1826]
Version: Nov 20 2017, 64 bit version
OS: Windows 7 64bit

Ok so here is a thing, when i try to assign some variable with string value i get this exception:
**Invalid src "value"**

Here is example:
```
set $newVariable , "some text"
Invalid src "some text"

```

**Comment by mrexodia** [comment]
That’s correct. There is no support for strings in the x64dbg command line...

**Comment by TheProxyRE** [comment]
Well scripting also fails
![image](https://user-images.githubusercontent.com/12687236/33390377-0ad3bf36-d536-11e7-975c-6190ad18b234.png)

**Comment by mrexodia** [comment]
Yes. You can find what the `mov` command does and does not support [here](http://help.x64dbg.com/en/latest/commands/general-purpose/mov.html).

**Comment by Ahmadmansoor** [comment]
@TheProxyRE 
pls check this plugin 
https://github.com/Ahmadmansoor/AdvancedScript
we're working on improving it

**Comment by Ahmadmansoor** [comment]
@TheProxyRE it supports now.
